### PR TITLE
Update composer to work with tinker 2.0

### PR DESCRIPTION
**Pull request by lucacri**
Tinker was just updated to 2.0. Nothing changed in the API so it will work with nova-tinker-tool too

### CODE REVIEW

**Comment by gaelreyrol** [comment]
Up

File: composer.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     ],
     "require": {
         "php": ">=7.1.0",
-        "laravel/tinker": "^1.0",
+        "laravel/tinker": "1 - 2",
```
Comment (ragingdave) [inline composer.json]: This looks like it's the "wrong" way to handle this from what I've seen.
pretty sure it's supposed to be `"^1.0|^2.0"`based on other repos requiring this same fix.

**Comment by mziraki** [comment]
@mpociot can u check this PR plz, I need this upgrade, tanx

**Comment by ragingdave** [comment]
is this package just not being maintained anymore? Tinker 2 has been available for quite some time, and for those trying to stay up to date it means we lost a valuable tool for development work, other than using the non nova based tinker web tool.

**Comment by mpociot** [comment]
Thank you for the PR!

**Pull request merged**